onnxruntime/core/framework/execution_frame.h

```diff
@@ -60,30 +62,6 @@ class ExecutionFrame {
                                                 const OrtAllocatorInfo& location,
                                                 const TensorShape& shape,
                                                 bool create_fence = false);
-
-  // ?? Cheng: What about non-tensor values??
-  // ?? Cheng: There are cases we may not want to use ORT_ENFORCE??
-  // ?? Cheng: Graph must be immutable for GetNodesInTopologicalOrder??
-  // Create tensor at index mlvalue, and allocate buffer for it.
-  // This tensor will own this buffer.
-  // This method is not thread safe!
-  Status AllocateTensorWithSelfOwnBuffer(int index,
-                                         MLDataType element_type,
-                                         const OrtAllocatorInfo& location,
-                                         const TensorShape& shape,
-                                         bool create_fence = false);
-
-  // Create tensor at index mlvalue, with pre-allocate buffer
-  // This tensor does not own the buffer.
-  // The executor / planner need to be careful about the
-  // lifetime of the buffer. Tensor itself won't manage it.
-  // This method is not thread safe!
-  Status AllocateTensorWithPreAllocateBuffer(int offset,
-                                             void* pBuffer,
-                                             MLDataType element_type,
-                                             const OrtAllocatorInfo& location,
-                                             const TensorShape& shape);
-
```

Comment:
These were only called from unit tests. Update unit tests (short term) to use AllocateMLValue... versions